radio/src/gui/128x64/startup_shutdown.cpp

```diff
@@ -35,10 +35,18 @@ void drawStartupAnimation(uint32_t duration, uint32_t totalDuration)
     return;
 
   uint8_t index = limit<uint8_t>(0, duration / (totalDuration / 5), 4);
-
   lcdRefreshWait();
   lcdClear();
 
+#if defined(RADIO_TPRO)
```

Comment:
This should be using the proper define for function switches rather than TPRO.

Reply:
In that vein, `NUM_FUNCTIONS_SWITCHES` gets rid of the magic number "6" also ;)